<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scouting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 22pt &quot;Verdana&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>613</y>
      <width>911</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextBrowser" name="saveFeedback">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>687</x>
      <y>477</y>
      <width>241</width>
      <height>117</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_15">
     <item>
      <widget class="QLabel" name="landRektLabel">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Totes Removed from Landfill</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QRadioButton" name="landfillYes">
         <property name="text">
          <string>Yes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="landfillNo">
         <property name="text">
          <string>No</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>195</y>
      <width>198</width>
      <height>278</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_14">
     <item>
      <widget class="QLabel" name="toteLabel">
       <property name="text">
        <string>Tote Stacking</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QRadioButton" name="toteStackYes">
         <property name="text">
          <string>Yes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="toteStackNo">
         <property name="text">
          <string>No</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="toteLevelLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Height&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="toteLevel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>687</x>
      <y>15</y>
      <width>197</width>
      <height>176</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="driveLabel">
       <property name="text">
        <string>Drive System</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="Mecanum">
         <property name="text">
          <string>Mecanum</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Tank">
         <property name="text">
          <string>Tank</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="tankWheels">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>196</y>
      <width>191</width>
      <height>127</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="binGrabLabel">
       <property name="text">
        <string>Bin Grabbing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="binGrabYes">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Yes</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="binGrabNo">
       <property name="text">
        <string>No</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>329</y>
      <width>171</width>
      <height>127</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="binStepLayout">
     <item>
      <widget class="QLabel" name="binStepLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Bin on Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="stepYes">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="stepNo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>462</y>
      <width>244</width>
      <height>143</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="binHeightLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Bin Stack Height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="binLevel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>14</y>
      <width>239</width>
      <height>176</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="liftTypeLabel">
       <property name="text">
        <string>Lift Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="forklift">
       <property name="text">
        <string>Forklift</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="conveyor">
       <property name="text">
        <string>Conveyor</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="customLift">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="liftText">
         <property name="placeholderText">
          <string>Other</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="commentsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>194</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Additional Comments</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>14</y>
      <width>416</width>
      <height>174</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QLabel" name="teamNumberLabel">
       <property name="text">
        <string>Team Number</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="teamForm">
         <property name="inputMask">
          <string>0000</string>
         </property>
         <property name="placeholderText">
          <string>####</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="blue">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="red">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="matchNumberLabel">
       <property name="text">
        <string>Match Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="matchForm">
       <property name="inputMask">
        <string>0000</string>
       </property>
       <property name="placeholderText">
        <string>##</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>416</width>
      <height>145</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="scoutNumberLabel">
         <property name="text">
          <string>Scout Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="scoutNumber">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="noodleGrabLabel">
         <property name="text">
          <string>Noodle Grab?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="noodleGrabYes">
         <property name="text">
          <string>Yes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="noodleGrabNo">
         <property name="text">
          <string>No</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>teamForm</tabstop>
  <tabstop>blue</tabstop>
  <tabstop>red</tabstop>
  <tabstop>matchForm</tabstop>
  <tabstop>forklift</tabstop>
  <tabstop>conveyor</tabstop>
  <tabstop>liftText</tabstop>
  <tabstop>binGrabYes</tabstop>
  <tabstop>binGrabNo</tabstop>
  <tabstop>stepYes</tabstop>
  <tabstop>stepNo</tabstop>
  <tabstop>binLevel</tabstop>
  <tabstop>Mecanum</tabstop>
  <tabstop>tankWheels</tabstop>
  <tabstop>toteStackYes</tabstop>
  <tabstop>toteStackNo</tabstop>
  <tabstop>toteLevel</tabstop>
  <tabstop>landfillYes</tabstop>
  <tabstop>landfillNo</tabstop>
  <tabstop>noodleGrabYes</tabstop>
  <tabstop>noodleGrabNo</tabstop>
  <tabstop>commentsBox</tabstop>
  <tabstop>save</tabstop>
  <tabstop>scoutNumber</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
